components/skus/browser/rs/lib/Cargo.toml

```diff
@@ -12,19 +12,19 @@ path = "tests/e2e.rs"
 required-features = ["e2e_test"]
 
 [dependencies]
-http = { version = "0.1" }
+http = { version = "0.2" }
 async-trait = "0.1.64"
 rand = { version = "0.7" }
 serde_json = "1.0"
 sha2 = "0.9"
 data-encoding = "2.1.2"
-tracing = { version = "0.1", default-features = false }
+tracing = { version = "0.1", default_features = false, features = ["attributes"] }
```

Comment:
This change will have no effect unless the feature is added to the BUILD.gn for the tracing crate. Are there any downstream effects of this on transitive dependencies? gnrt can figure that out, but it's easier if we can avoid running it because it's kind of a pita